### PR TITLE
Add support for .NET Framework 4.5.2 

### PR DESCRIPTION
**Pull request by Guad**
Simple change requested by #10 
Tested this on another project of mine and worked fine.

### CODE REVIEW

File: src/ReadLine/project.json [hunk 1]
```diff
@@ -26,6 +26,8 @@
         "NETStandard.Library": "1.6.1",
         "System.Console": "4.3.0"
       }
+    },
+    "net452": {
```
Comment (tonerdo) [inline src/ReadLine/project.json]: Let's make this `net45` to support a broader range. Also the curly braces don't need to be on separate lines

**Review by tonerdo — APPROVED** [review]
That was quick! Thanks @Guad, just a single comment but apart from that this looks good.

File: src/ReadLine/project.json (hunk 1 above)
Comment (tonerdo) [reply src/ReadLine/project.json]: Kindly re-test after you make the change just to be sure

**Comment by Guad** [comment]
Now that you mention it, using `net45` makes `dotnet pack` spew an omnious `Object reference not set to an instance of an object.`.
Changing it to net452 fixes it

**Comment by tonerdo** [comment]
@Guad run a `dotnet restore` before `dotnet pack` it'll get rid of the error

**Comment by Guad** [comment]
@tsolarin Right, works fine then.

**Comment by tonerdo** [comment]
@Guad Thank you!!! This will be going out in the next release
/cc @sergiorykov

**Pull request merged**